ssn/integrated/oldssn.ttl

```diff
@@ -822,20 +804,6 @@ ssn:Process rdf:type owl:Class ;
             rdfs:seeAlso "http://www.w3.org/2005/Incubator/ssn/wiki/SSN_Model#Process" .
 
 
-###  http://www.w3.org/ns/ssn/Property
```

Comment:
removed ssn:Property in the oldssn.ttl, according to the proposed integration methodology